dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/StreamingChatContent.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.SemanticKernel.AI.ChatCompletion;
+
+/// <summary>
+/// Streaming chat result update.
+/// </summary>
+public abstract class StreamingChatContent : StreamingContent
```

Comment:
Is it accurate to say this will `Represent part of response from a chat completion when using streaming`?

If yes, then I'm not sure this naming fits because `StreamingChatContent` conveys that I can use this to get the streaming chat content but it really represents the latest chunk that has already been streamed.

I think `StreamedChatMessageContent` would be better.

Reply:
Yeah, it does make sense I'm following the naming Azure SDK used for its classes.

Here: [StreamingChatCompletionsUpdate.cs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/openai/Azure.AI.OpenAI/src/Custom/StreamingChatCompletionsUpdate.cs)